autodispose/src/main/java/com/uber/autodispose/ScopeUtil.java

```diff
@@ -74,15 +75,25 @@ private ScopeUtil() {
       @Override public MaybeSource<? extends LifecycleEndNotification> call() throws Exception {
         E lastEvent = provider.peekLifecycle();
         if (checkStartBoundary && lastEvent == null) {
-          throw new LifecycleNotStartedException();
+          LifecycleNotStartedException exception = new LifecycleNotStartedException();
+          if (AutoDisposePlugins.outsideLifecycleHandler != null) {
+            AutoDisposePlugins.outsideLifecycleHandler.accept(exception);
```

Comment:
Also shouldn't we do something based if we proceed past accept? Like dispose or something?